network/stream/v2/snapshot_sync_test.go

```diff
@@ -68,7 +68,7 @@ func TestSyncingViaGlobalSync(t *testing.T) {
 		log.Info(fmt.Sprintf("Running test with %d chunks and %d nodes...", *chunks, *nodes))
 		testSyncingViaGlobalSync(t, *chunks, *nodes)
 	} else {
-		chunkCounts := []int{4, 32}
```

Comment:
We can't sync 32 chunks ? :)

Reply:
It times out on travis. :) And it actually does not even flake locally. The only thing that I managed to see is the goroutine leak in hive that is fixed. If it is flaky even with only 4 chunks on travis, we should disable this test and dedicate more time to find the root cause.